crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -91,6 +91,13 @@ Examples:
         env = "PATHFINDER_SQLITE_WAL", 
     )]
     sqlite_wal: bool,
+
+    #[arg(
+        long = "max-connections",
+        long_help = "Set the maximum number of connections allowed. Default is 1024.",
+        env = "PATHFINDER_MAX_CONNECTIONS"
+    )]
+    max_connections: Option<u32>,
```

Comment:
Should we just set the default within the config already?
```suggestion

    #[arg(
        long = "max-rpc-connections",
        long_help = "Set the maximum number of RPC connections allowed.",
        env = "PATHFINDER_MAX_RPC_CONNECTIONS",
        default_value = 1024
    )]
    max_connections: u32,
```

Reply:
Done